fdbrpc/FlowTransport.actor.cpp

```diff
@@ -1967,3 +1992,60 @@ void FlowTransport::removePublicKey(StringRef name) {
 void FlowTransport::removeAllPublicKeys() {
 	self->publicKeys.clear();
 }
+
+void FlowTransport::loadPublicKeyFile(const std::string& filePath) {
+	if (!fileExists(filePath)) {
+		throw file_not_found();
+	}
+	int64_t const len = fileSize(filePath);
+	if (len <= 0) {
+		TraceEvent(SevWarn, "AuthzPublicKeySetEmpty").detail("Path", filePath);
+	} else if (len > FLOW_KNOBS->PUBLIC_KEY_FILE_MAX_SIZE) {
+		throw file_too_large();
+	} else {
+		auto json = readFileBytes(filePath, len);
+		self->applyPublicKeySet(StringRef(json));
+	}
+}
+
+ACTOR static Future<Void> watchPublicKeyJwksFile(std::string filePath, TransportData* self) {
+	state AsyncTrigger fileChanged;
+	state Future<Void> fileWatch;
+	state unsigned errorCount = 0; // error since watch start or last successful refresh
+	loop {
```

Comment:
This loop is a bit weird. I guess this is there so we don't have a dependency between IAsyncFileSystem initialization and FlowTransport. Can you please add a comment explaining why this is here?